src/connected/AppBar.tsx

```diff
@@ -29,6 +30,7 @@ const mapDispatchToProps = {};
 const styles = (theme: Theme) => createStyles(
     {
         toolbar: {
+            backgroundColor: 'white',
```

Comment:
Use VIEWER_TOOLBAR_COLOR in config.tx, by default it is null, so primary theme color is used.